R/pipe.R

```diff
@@ -22,26 +22,11 @@ use_pipe <- function(export = TRUE) {
   check_is_package("use_pipe()")
   check_uses_roxygen("use_pipe()")
 
-  use_dependency("magrittr", "Imports")
-
   if (export) {
+    use_dependency("magrittr", "Imports")
```

Comment:
Shouldn't magrittr always be added to imports?

Reply:
Yes, but `use_import_from()` will run it if needed.